### ISSUE TITLE
Enhance the Keyboard app to include Select All, Copy, Paste function buttons in Keyboard to support Android TV

### ISSUE THREAD

**Issue by Jonnathanjons** [yuliskov/LeanKeyKeyboard#59]
Can you please enhance the LeanKey Keyboard to include functions such as Select All or  Select row ,  Copy, Paste from Clipboard , Go back buttons within the Keyboard so that Google TV users can utilize this feature. Currently this keyboard "Paste from Clipboard" button but  there is no  button to Copy to Clipboard  or Select specific rows etc. It will be really helpful. Awaiting the update.

**Comment by basxto** [comment]
I love the "Paste from Clipboard" and use it when I copy the clipboard from other devices. But I don’t think selecting text is possible in a meaningful way. So it should probably be more a "Copy all"

**Comment by Jonnathanjons** [comment]
I agree , at least I expect there should be an option for "select all" and "Copy" function in the new version of this keyboard. Its just another 2 buttons..but its currently not available in any keyboards including gboard. btw I replaced Gboard with LeanKeyboard  in all my devices only for Copy to Clipboard functionality. unfortunately there should be a supporting version for firestick aswell. thanks a awaiting the updates..

**Comment by throwarray** [comment]
You can send keyevent 278 for copy, 277 for cut. Ctrl+A 113 29...
A full keyboard layout that's much smaller and an OLED theme would be nice too.

**Comment by yuliskov** [comment]
thanks for the feedback, friends

Regards, Yuriy

**Comment by 43Rageon** [comment]
Ah, I started a similar topic without seeing this one, sorry about that.

I agree with what OP wrote, it's essential for LeanKeyboard to have these features. Why haven't they been added since 2023?
